### DISCUSSION TITLE
REST API - retrieve all the content related to a menu-alias

### DISCUSSION THREAD

**Discussion by bellu** [joomla/joomla-cms#47010] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
I'm trying to use Joomla REST API for the backend and use a JS framework to build the frontend.
Looking at the REST API, it seems to be tricky to understand the content of a menu-alias.
For example, how can i know all the modules, content ecc... associated to a menu alias?
At the moment, i have to loop all the modules and filter it for a menu alias every time.

### Describe the solution you'd like
I think could be useful, to have an endpoint where given a menu-alias i can retrieve everything associated to that alias.
Like article, modules ecc...

**Comment by Gunjan-Rajput** [comment]
I want to work on this issue.
Please assigned me.

**Comment by alikon** [comment]
there is no need to assign, please work on it and submit a pull request

p.s
you can even join here https://joomlacommunity.cloud.mattermost.com/main/channels/gsoc2023 if you are proposing yourself as a GSOC23 student
